### DISCUSSION TITLE
Enhancement: Separate both character's vocals into multiple audio files

### DISCUSSION THREAD

**Discussion by treacherousfiend** [FunkinCrew/Funkin#1618] [Ideas]
## What is your suggestion, and why should it be implemented?
Currently the vocals for each song are all in one file, which is fine in most scenarios, but in scenarios where both characters are singing at once, a missed note causes both of them to stutter.

Separating the audio tracks would allow the other character to keep going instead of oddly stuttering with the player.
for file space reasons, only songs where both characters sing at the same time would need these separate audio tracks, while the songs without them can continue to be one vocal track.

As a side note, it may be that for some players the feedback of both characters cutting out when they miss a note is better/more noticeable than only one character cutting out, maybe this could be a toggleable option?
Ideally with the option for both characters stuttering wouldn't need to keep the unified vocal track and just mute both audio tracks.

**Comment by Benefonde** [comment]
yes, I want this

**Comment by ghost** [comment]
You would have to ask Ninjamuffin on that one.
EDIT: Sorry, I thought this was KadeEngine
